flows/load_image.py

```diff
@@ -84,6 +85,9 @@ def load_image(FILENAME):
 		image.image = np.asarray(hdul[0].data, dtype='float64')
 		image.shape = image.image.shape
 
+		image.head = hdr
```

Comment:
Let's rename this to "header" instead of "head"

Reply:
Done